cmd/internal/server/core.go

```diff
@@ -23,12 +24,19 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-var InternalPlugins = map[string]plugin.Root{
-	"aws":        &aws.Root{},
-	"docker":     &docker.Root{},
-	"gcp":        &gcp.Root{},
-	"kubernetes": &kubernetes.Root{},
-}
+// InternalPlugins lists the plugins enabled by default in Wash.
+var InternalPlugins = func() map[string]plugin.Root {
+	plugins := map[string]plugin.Root{
+		"aws":        &aws.Root{},
+		"docker":     &docker.Root{},
+		"gcp":        &gcp.Root{},
+		"kubernetes": &kubernetes.Root{},
+	}
+	if os.Getenv("WASH_LOCALFS") != "" {
```

Comment:
Now that https://github.com/puppetlabs/wash/issues/455 is resolved, I think this should move over to https://github.com/puppetlabs/wash/blob/master/cmd/server.go#L137. That way, you can just say "You can enable localfs by setting WASH_LOCALFS to true" or something w/o having people mess with their `plugins` key.